mathjax3-ts/output/chtml/fonts/tex.ts

```diff
@@ -414,12 +428,45 @@ export class TeXFont extends FontData {
         const data = this.getChar('normal', n) || this.getChar('-size4', n);
         const css: StyleData = {content: '"' + this.char(n, true) + '"'};
         if (part !== 'ext') {
-            css.padding = this.em(data[0]) + ' 0 ' + this.em(data[1]);
+            css.padding = this.em0(data[0]) + ' 0 ' + this.em0(data[1]);
         }
         styles['.MJX-TEX mjx-stretchy-v[c="' + c + '"] mjx-' + part + ' mjx-c::before'] = css;
         return data[0] + data[1];
     }
 
+    /*
+     * @param{StyleList} styles    The style object to add styles to
+     * @param{string} c            The delimiter character string
+     * @param{DelimiterData} data  The data for the delimiter whose CSS is to be added
+     */
+    protected addDelimiterHStyles(styles: StyleList, c: string, data: DelimiterData) {
+        this.addDelimiterHPart(styles, c, 'beg', data.stretch[0]);
+        this.addDelimiterHPart(styles, c, 'ext', data.stretch[1], !(data.stretch[0] || data.stretch[2]));
+        this.addDelimiterHPart(styles, c, 'end', data.stretch[2]);
+        if (data.stretch[3]) {
+            this.addDelimiterHPart(styles, c, 'mid', data.stretch[3]);
+            styles['.MJX-TEX mjx-stretchy-h[c="' + c + '"] > mjx-ext'] = {width: '50%'}
+        }
+    }
+
+    /*
+     * @param{StyleList} styles  The style object to add styles to
+     * @param{string} c          The vertical character whose part is being added
+     * @param{string} part       The name of the part (beg, ext, end, mid) that is being added
+     * @param{number} n          The unicode character to use for the part
+     */
+    protected addDelimiterHPart(styles: StyleList, c: string, part: string, n: number, force: boolean = false) {
+        if (!n) return 0;
```

Comment:
Missing braces.

Reply:
I thought we agreed that returns could be on the same line.